packages/identity/config/tsconfig.browser.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../config/tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "../dist/browser",
```

Comment:
Totally agree and open to discussion.

IMO we should adopt the same policy as `sdk-js`:
- No build-in support fot IE (`"target": "es6"// aka ES2015`)
- Add polyfill for IE tests and in UMD

for now i'll use recommendations from https://github.com/tsconfig/bases to select correct targets

Reply:
Little update. We must target ES5 for our code to run on IE11. Some unavailable syntax is used otherwise (ie. class, arrow-functions) and it is not possible to polyfill those.

Reply:
here is something that can help https://webhint.io/docs/user-guide/hints/hint-typescript-config/target/. They give a mapping between ECMAScript and browser versions

Another interesting thread about a proposal to use engine/browser versions instead of the target ECMAScript https://github.com/microsoft/TypeScript/issues/19183
This [comment](https://github.com/microsoft/TypeScript/issues/19183#issuecomment-450440200) seems to solve our issue without much complexity

Reply:
The solution proposed in the commit I linked works fine. But It only tests our code (we could probably run it on a umd version to tests our dependencies too).